src/adhocracy_core/docs/deletion.rst

```diff
@@ -1,123 +1,123 @@
+  parameter *include=hidden* can be used to include hidden resources in
+  pool listings and other search queries.  If its value is *hidden*,
+  resources will be found regardless of the value of their *hidden*
+  flag.  However, only those with *hide_resource* permission are ever
+  able to view the contents of hidden resources.  It's also possible to
+  set *include=visible* to get only non-deleted and non-hidden
+  resources, but it's not necessary since that is the default.
+* If the frontend attempts to retrieve a *hidden* resource via GET, the
+  backend normally responds with HTTP status code *410 Gone*.
+  FIXME Not implemented yet, since the frontend doesn't yet need it: The
+  frontend can override this by adding the parameter *include=hidden* to
+  the GET request, just as in search queries.  Managers (those with
+  *hide_resource* permission) can view hidden resources in this way.
+  Those without this permission will still get a *410 Gone* if the
+  resource is hidden.
+* The body of the *410 Gone* is a small JSON document that explains why
+  the resource is gone (for future use if there may be other reasons
+  than *hidden*). It also shows who made the last change to the resource
+  and when::
+
+      { "reason": "hidden",
         "modified_by:" "<path-to-user>",
         "modification_date": "<timestamp>"}
 
-  Often the last modification will have been the deletion or hiding of
-  the resource, but there is no guarantee that this is always the case.
-  Especially, the resource may be marked as hidden/deleted because one of its
-  ancestors was hidden/deleted (as that status is inherited). In that case,
-  the person who last modified the child resource likely has nothing to do
-  with the person who hid/deleted the ancestor resource.
-* *Deleted* or *hidden* resources may still be referenced from other
-  resources. If the frontend follows such references it must therefore
-  be prepared to encounter *410 Gone* responses and deal with them
-  appropriately (e.g. by silently skipping them or by showing an
-  explanation such as "Comment deleted").
+  Often the last modification will have been the hiding of the resource,
+  but there is no guarantee that this is always the case.  Especially,
+  the resource may be marked as hidden because one of its ancestors was
+  hidden (as that status is inherited). In that case, the person who
+  last modified the child resource likely has nothing to do with the
+  person who hid the ancestor resource.
+* *Hidden* resources may still be referenced from other resources. If
+  the frontend follows such references it must therefore be prepared to
+  encounter *410 Gone* responses and deal with them appropriately (e.g.
+  by silently skipping them or by showing an explanation such as
+  "Comment deleted").
 * FIXME Not implemented yet, since the frontend doesn't yet need it.
-  *Deleted* and *hidden* resources will normally not be shown in
```

Comment:
cleanup not related to this pull request: can be removed to cleanup fixmes we won't fix or are part of the refactoring list